arbnode/blockmetadata.go

```diff
@@ -256,3 +258,10 @@ func (b *BlockMetadataFetcher) StopAndWait() {
 	b.StopWaiter.StopAndWait()
 	b.client.Close()
 }
+
```

Comment:
the indentation of this seems off from the official fork: https://github.com/OffchainLabs/nitro/blob/v3.6.7/arbnode/blockmetadata.go